tests/Vesting.test.ts

```diff
@@ -104,43 +104,70 @@ describe("Vesting service", async () => {
 
 		});
 		it("lock as sasha and claim as pavel", async (context: localTC) => {
-		    const {h, h: { network, actors, delay, state }} = context;
+			const {h, h: { network, actors, delay, state }} = context;
 			const { sasha, tom, pavel } = actors;
 
+			const txSplit = new Tx();
+			const sashaMoney = await sasha.utxos;
+
+			txSplit.addInput(sashaMoney[0]);
+			txSplit.addOutput(new TxOutput(sasha.address, new Value(3n * ADA)));
+			txSplit.addOutput(new TxOutput(sasha.address, new Value(3n * ADA)));
+			txSplit.addOutput(
+			    new TxOutput(
+			        sasha.address,
+			        new Value(sashaMoney[0].value.lovelace - 10n * ADA)
+			    )
+			);
+
+			await h.submitTx(txSplit);
+
 			const v = new Vesting(context);
 			const t = BigInt(Date.now());
-			const d = t + BigInt(2*60*60*1000);
+			const deadline = t + BigInt(2*60*60*1000); // now + two hours
 
 			const tcx = await v.mkTxnDepositValueForVesting({
-				sponsor: sasha,   // need sasha  
+				sponsor: sasha,   
 				payee: pavel.address, // maybe pkh? 
-				deadline: d
+				deadline: deadline
 			});
 
+			const txId = await h.submitTx(tcx.tx, "force");
 			// explore the transaction data:
 			expect(tcx.inputs[0].origOutput.value.lovelace).toBeTypeOf('bigint');
-			expect(tcx.inputs[1].origOutput.value.lovelace).toBe(5000000n);
+			expect(tcx.inputs[1].origOutput.value.lovelace).toBeTypeOf('bigint');
 			expect(tcx.outputs[0].datum.data.toSchemaJson().length).toBe(175);
-
-			const txId = await h.submitTx(tcx.tx, "force");
+			expect(tcx.outputs[0].datum.data.list[2].value).toBe(deadline);
 
 			expect((txId.hex).length).toBe(64);
-			expect((await pavel.utxos).length).toBe(2);
+			expect((await sasha.utxos).length).toBeGreaterThan(1); //mkTxnClaim uses 2 utxos
 
 			const validatorAddress = Address.fromValidatorHash(v.compiledContract.validatorHash)
 			const valUtxos = await network.getUtxos(validatorAddress)
+			
+			// check currentSlot:
+			expect(currentSlot()).toBe();
```

Comment:
StellarTestHelper.currentSlot()

Reply:
https://github.com/aleeusgr/stellar-vesting/blob/1780311fb86af711caec8727ad9f734c0e756f40/lib/StellarTestHelper.ts#L125

Reply:
https://github.com/aleeusgr/stellar-vesting/blob/1780311fb86af711caec8727ad9f734c0e756f40/tests/Vesting.test.ts#L43

Reply:
Vesting test helper is available via `h`?

Reply:
StellarCapoTestHelper

Why Capo? Capo is a design pattern

Reply:
StellarCapoTestHelper extends StellarTestHelper, L376 in /lib/StellarTestHelper.ts

Reply:
1691477246n

Reply:
35m ☯️;